src/test/java/org/opentripplanner/middleware/ApiKeyManagementTest.java

```diff
@@ -47,7 +44,6 @@
     @BeforeAll
     public static void setUp() throws IOException {
         OtpMiddlewareTest.setUp();
```

Comment:
Needs a `assumeTrue(getBooleanEnvVar("RUN_E2E"));` on the first line of this method so that this only runs during E2E testing.

Reply:
Updated.